src/Residue.php

```diff
@@ -40,10 +39,83 @@ public static function create(float $value): self
         return new self($value);
     }
 
+    /**
+     * f(N, Dv, Dc, S) with :
+     *  - N the number
+     *  - Dv the divider
+     *  - Dc the number of decimals
+     *  - S the step size.
+     *
+     * This function is described by this equations :
+     * [
+     *  - S = S || 10^(-Dc)
+     *  - Dc = Dc || GetNumberOfDecimalOfStep(S)
+     *
+     *  - rm + p¹ + p² + ... p^n = N (where rm the reminder, p a part and n in range [1, Dv])
```

Comment:
reminder => remainder 

I hesitate between `remainder` and `leftover` _(adj. remaining/surplus)_ I don't know which is the best

Reply:
> reminder => remainder

Yeah a bit of typo, well spoted. It is in the sense that I need to be reminded that there is a left over... :wink: 

Both are understandable but remainder is more accurate in mathematical context. I mind it was the good choice.